src/MatBlazor.Demo/Demo/DemoMatSlider.razor

```diff
@@ -23,7 +23,7 @@
 
     </Content>
     <SourceContent>
-    	<BlazorFiddle Template="MatBlazor" Code=@(@"
+        <BlazorFiddle Template="MatBlazor" Code=@(@"
```

Comment:
Generally, adding tabulation or other stuff to a PR that doesn't change a whole lot can add noise to the PR. I would generally split up the PR so we are aware that one is simply to clean up the code styling, while the other is to fix/enhance the code

Reply:
reverted. my IDE didn't show whitespaces and autoformatted it for me